documentation/LLC-quickstart.md

```diff
@@ -0,0 +1,401 @@
+Getting Started - Generate the RLC rest-level client libraries.
+================================================================
+# Prerequisites
+You may refer to this [link](https://github.com/Azure/azure-sdk-for-js/blob/main/CONTRIBUTING.md#prerequisites) for the environment set up prerequisites in azure-sdk-for-js repository. 
+
+# How to generate RLC
+We are working on to automatically generate everything right now, but currently we still need some manual work to get a releasable package. Here're the steps of how to get the package.
+1. **Create a swagger/README.md file.**  
```

Comment:
Where do I need to create the swagger/README.md? Reading from top to bottom, it would look like it has to be in the root folder